modules/nw-egress-router-about.adoc

```diff
@@ -134,36 +142,38 @@ spec:
       containers:
         ...
 ----
+<1> Ensure that replicas is set to `1`, because only one pod can use a given egress source IP address at any time. This means that only a single copy of the router runs on a node.
+
+<2> Specify the `Pod` object template for the egress router pod.
 endif::openshift-sdn[]
 
 ifdef::ovn[]
+To avoid downtime, the Cluster Network Operator deploys the egress router pod as a deployment resource. The deployment name is `egress-router-cni-deployment`.  The pod that corresponds to the deployment has a label of `app=egress-router-cni`.
```

Comment:
```suggestion
To avoid downtime, the Cluster Network Operator deploys the egress router pod as a deployment resource. The deployment name is `egress-router-cni-deployment`. The pod that corresponds to the deployment has a label of `app=egress-router-cni`.
```